File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task35_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>
#include <vector>

class BusPassengers {
public:
    static int countPassengers(std::vector<std::vector<int>>& stops) {
        int passengers = 0;
        for (const auto& stop : stops) {
            passengers += stop[0] - stop[1];
        }
        return passengers;
    }
};

int main() {
    std::vector<std::vector<int>> stops = {{10,0}, {3,5}, {5,8}};
    std::cout << BusPassengers::countPassengers(stops) << std::endl;
    return 0;
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task35_CLAUDE_claude-3-5-sonnet-20240620.java

public class BusPassengers {
    public static int countPassengers(int[][] stops) {
        int passengers = 0;
        for (int[] stop : stops) {
            passengers += stop[0] - stop[1];
        }
        return passengers;
    }

    public static void main(String[] args) {
        int[][] stops = {{10,0}, {3,5}, {5,8}};
        System.out.println(countPassengers(stops));
    }
}
